programs/local/LocalServer.cpp

```diff
@@ -575,7 +593,8 @@ void LocalServer::processConfig()
       * Otherwise, metadata of temporary File(format, EXPLICIT_PATH) tables will pollute metadata/ directory;
       *  if such tables will not be dropped, clickhouse-server will not be able to load them due to security reasons.
       */
-    std::string default_database = config().getString("default_database", "_local");
+    std::string default_database = config().getString("default_database", "");
+    default_database = default_database + "_local";
```

Comment:
hmm,  The cases when current_database is default. Since default database is already exist, it throws error " default database is already exists"?
 It fails with default database exists error. Since Clickhouse-local already creates default daItabase as part of DatabaseMemory and try to create database again here.
https://github.com/ClickHouse/ClickHouse/blob/master/src/Interpreters/loadMetadata.cpp#L89
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/86018844/180620125-a3a2cad6-e00d-4493-bb7b-8760bcd08acf.png">